build.sbt

```diff
@@ -20,10 +20,7 @@ val common = Seq(
 
   libraryDependencies ++= Seq(
     "io.dylemma"                  %% "xml-spac"              % "0.3",
-    "org.apache.hadoop"           %  "hadoop-aws"            % "2.8.1"
-      exclude("javax.servlet",     "servlet-api")
-      exclude("javax.servlet.jsp", "jsp-api")
-      exclude("org.mortbay.jetty", "servlet-api"),
```

Comment:
These were explicit excludes to get Spark UI to work locally. We should test if it still works.